CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
+## 0.11.0 / 2015-07-27
+
+* [FEATURE] Add stats from /proc/net/snmp.
+* [FEATURE] Add support for FreeBSD.
```

Comment:
Gah - I really should have made sure that the commits for this one were squashed before merging. Next time.